src/Resources/config/app/config.yaml

```
@@ -18,3 +19,9 @@ jms_serializer:
             setono-sylius-gift-card-sylius:
                 namespace_prefix: "Setono\\SyliusGiftCardPlugin\\Model"
                 path: "@SetonoSyliusGiftCardPlugin/Resources/config/serializer"
+
+liip_imagine:
+    filter_sets:
+        gift_card_pdf:
```

```suggestion
        setono_sylius_gift_card_background:
```

To be consistent with Sylius I think the namespace should be included. I also renamed to `background`. Wdyt?

---

Yes, good point